File: accidentes/FicheroAccidentes-Java-IvanRC/src/main/java/dto/distrito/DistritoDto.java
package dto.distrito;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Root(name = "distrito")
public class DistritoDto {
    @Element(name = "id_distrito")
    String id; // 5
    @Element(name = "nombre")
    String nombre; // 6
}

File: accidentes/FicheroAccidentes-Kotlin-IvanRC/src/main/kotlin/dto/distrito/DistritoDto.kt
package dto.distrito

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "distrito")
data class DistritoDto (
    @field:Element(name = "id_distrito")
    @param:Element(name = "id_distrito")
    val id: String, // 5
    @field:Element(name = "nombre")
    @param:Element(name = "nombre")
    val nombre: String // 6
)
